src/button/index.ts

```diff
@@ -35,7 +36,7 @@ export interface ButtonProperties extends ThemedProperties, InputEventProperties
 	onClick?(): void;
 }
 
-export const ThemedBase = ThemedMixin(WidgetBase);
+export const ThemedBase = ThemedMixin(FocusMixin(WidgetBase));
```

Comment:
Not really `ThemedBase` anymore 😃

Reply:
Hah, I never even noticed it was called that 😂